### PR TITLE
Ran 'npm audit fix' to fix packages.

### PR DESCRIPTION
**Pull request by uriyang**
- Vulnerabilities in jsrsasign, webpack

### CODE REVIEW

**Comment by uriyang** [comment]
signed, please check my pr

**Comment by uriyang** [comment]
I modified it.

File: package.json [hunk 1]
```diff
@@ -53,8 +53,8 @@
     "jquery-contextmenu": "2.4.5",
     "jquery-i18next": "1.2.1",
     "js-md5": "0.6.1",
-    "js-utils": "github:jitsi/js-utils#cf11996bd866fdb47326c59a5d3bc24be17282d4",
-    "jsrsasign": "8.0.12",
+    "js-utils": "github:jitsi/js-utils#df68966e3c65b5c57fcd2670da1326a2c77518d1",
```
Comment (saghul) [inline package.json:56]: please undo this line

File: package.json [hunk 2]
```diff
@@ -53,8 +53,8 @@
...
+    "jsrsasign": "^8.0.15",
```
Comment (saghul) [inline package.json:57]: please remove the caret, we pin dependencies to the exact version

File: package.json [hunk 3]
```diff
@@ -127,7 +127,7 @@
     "string-replace-loader": "2.1.1",
     "style-loader": "0.19.0",
     "unorm": "1.6.0",
-    "webpack": "4.27.1",
+    "webpack": "^4.42.1",
```
Comment (saghul) [inline package.json:130]: ditto

File: package.json (hunk 1 above)
Comment (uriyang) [reply package.json]: I modified.

File: package.json (hunk 2 above)
Comment (uriyang) [reply package.json]: I modified.

File: package.json (hunk 3 above)
Comment (uriyang) [reply package.json]: I modified.

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by luixxiul** [comment]
Deprecated by https://github.com/jitsi/jitsi-meet/pull/6773

**Pull request closed without merging**